packages/studioCMS/src/utils/contentHelper.ts

```diff
@@ -86,6 +88,30 @@ export async function contentHelper(
     return { ...pageData, content: pageContent.content||"Failed to fetch Content" };
 }
 
+export async function getPageById(id: string): Promise<ContentHelperTempResponse> {
+    const pageData = await db
+        .select().from(PageData)
+        .where(eq(PageData.id, id))
+        .get();
+
+    if(!pageData) {
+        return {} as ContentHelperTempResponse;
+    }
+
+    const LangToGet = "default";
```

Comment:
```suggestion
    const langToGet = "default";
```

Reply:
Actually on this as well, is the `pageContent` query meant to use `langToGet` somewhere? At the moment it isn't being used

Reply:
its for Future i18n compatability.

Reply:
[3686430](https://github.com/astrolicious/studiocms/pull/142/commits/3686430b99ddb1a9be1cabd4f68c8ab9f6bfe913) renamed.... lol next time please change all occurrences...